src/components/form/range/range.tsx

```diff
@@ -116,6 +116,24 @@ export class EuiRangeClass extends Component<
     });
   };
 
+  handleAriaValueText = (
+    ticks: EuiRangeTick[],
+    currentVal: string | number
+  ): string | undefined => {
+    const target = ticks.find(
+      (tick) => tick.value.toString() === currentVal.toString()
+    );
+
+    if (target) {
+      return target.accessibleLabel
+        ? `${target.value}, (${target.accessibleLabel})`
+        // Fall back to the label if it's a usable string
+        : typeof target.label === 'string'
```

Comment:
Gah sorry, Prettier is unhappy with this
```suggestion
        : typeof target.label === 'string' // Fall back to the label if it's a usable string
```